Fragment path names (#44)
NCAS-CMS/cfa-conventions: source/cfa.md

  which may be fully qualified URLs, containing the fragments. Each
  value identifies the external resource which contains the fragment.

  The URI may contain a string substitution defined by the **`substitutions`** attribute of the `file` variable.
  This attribute takes a string value comprising blank-separated elements of the form "`base: substitution`", where `base` is a case-sensitive keyword that defines the part of a file URI which is to be replaced by the string defined by `substitution`.
  The value of `base` must have the form `${...}`, where `...` represents one or more letters, digits, and underscores.
  The order of elements is not significant.
  The use of substitutions can save space in the file and, if the fragment files were to be moved, provides a means of updating the CFA-netCDF file without having to access the `file` variable's data. 

>   A URI must be a fully qualified URL, a file URI, or else a file name whose path is relative to the location of the CFA-netCDF file.

Reviewer: ```suggestion
  A URI must be a fully qualified URL, a file URI, or else a file path which is relative to the location of the CFA-netCDF file.
```

would be clearer to me, since the initial use of 'name' could imply no path required?

*An aggregated data variable whose aggregated data comprises two
fragments. Each fragment spans half of the aggregated `time` dimension
and the whole of the other three aggregated dimensions, and is stored
in an external netCDF file in a variable call `temp`. The fragment

> URIs define file loctions relative to the CFA-netCDF file. Both

Reviewer: ```suggestion
URIs define file locations relative to the CFA-netCDF file. Both
```
